Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/services/impl/AuditServiceImpl.java

```diff
@@ -213,8 +207,9 @@ private String marshallAuditMessage(AuditMessageType mess) {
         if (mess != null) {
             try {
                 JAXBContextHandler oHandler = new JAXBContextHandler();
-                JAXBContext jc = oHandler.getJAXBContext("com.services.nhinc.schema.auditmessage");
+                JAXBContext jc = oHandler.getJAXBContext(AUDIT_MESSAGE_NAMESPACE);
```

Comment:
You can do something like new JAXBContextHandler().getJAXBContext(AUDIT_MESSAGE_NAMESPACE); instead of creating JAXBContextHandler object